tests/textDocument/formatting.rkt

```diff
@@ -21,7 +21,8 @@ END
       (let ([notif (make-notification
                     "textDocument/didChange"
                     (hasheq 'textDocument
-                            (hasheq 'uri uri)
+                            (hasheq 'uri uri
+                                    'version 0)
```

Comment:
How do we decide this number? According to spec

> The version number of this document (it will increase after each change, including undo/redo).

Seems it need to count?

Reply:
Yes. But the version number is sent by the client. We don't need to concentrate it. We only need to make sure the version of a new `build-trace%` match its text version.

Reply:
I see.